src/wolfgame.py

```diff
@@ -47,6 +46,7 @@
 from src.utilities import *
 from src import db, decorators, events, logger, proxy, debuglog, errlog, plog
 from src.messages import messages
+from src.warnings import *
```

Comment:
Not too warm on a blanket import here; we do it for utilities but I don't care as much about that ("Just give me the utility functions"). It doesn't matter that much though (especially since you defined `__all__`) so I'm not going to hold up merging if that's the only issue left.